test/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter_test.cc

```diff
@@ -335,6 +335,76 @@ TEST_F(GrpcJsonTranscoderConfigTest, InvalidVariableBinding) {
   EXPECT_FALSE(transcoder);
 }
 
+// of path segment and can capture it in a wildcard segment.
+TEST_F(GrpcJsonTranscoderConfigTest, UnregisteredCustomVerb) {
+  JsonTranscoderConfig config(
+      getProtoConfig(TestEnvironment::runfilesPath("test/proto/bookstore.descriptor"),
+                     "bookstore.Bookstore", false),
+      *api_);
+
+  // It is matched to PostWildcard `POST /wildcard/{arg=**}`.
+  Http::TestRequestHeaderMapImpl headers{{":method", "POST"},
```

Comment:
":unknown" was not treated as custom verb,  template doesn't require a custom verb so it matched